hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -182,12 +196,13 @@
   inter_module: {
     'connect': {
       'flash_ctrl.flash': ['eflash.flash_ctrl']
+      'pwrmgr.pwr_rst'  : ['rstmgr.pwr'],
     }
 
     // top is to connect to top net/struct.
     // It defines the signal in the top and connect from the module,
     // use of the signal is up to top template
-    'top': [],
+    'top': ['rstmgr.resets'],
```

Comment:
@eunchan 
this is the right way to do this right? I originally thought i needed a similar structure to `connect`, but seems that is not the case.

Reply:
right. Thanks for adding the example :)

Reply:
So, do you like the overall inter-module connection scheme? feel free to ping me if you think something is awkward.

Reply:
yeah i think this is good. The only part that i wonder about is what happens if we ever have a signal that needs to go to two different places, but one of these is more `uni` in nature, and the other is more of a `req_rsp`.  I guess in that case we just let the signal go out from a module in two places right? Doesn't seem like it cost us anything, and most likely synthesis will simplify it anyways.